content/reference/endpoints/rental_contacts.md

```diff
@@ -0,0 +1,108 @@
+roles                           | [Array of Strings](/reference/enums#rental-contact-roles) | Read | Rental contact's roles
+--------------------------------|----------|------------|--------------
+created_at                      | [Time](/reference/enums#formats) | Read         | Rental's create time.
+updated_at                      | [Time](/reference/enums#formats) | Read         | Rental's update time.
+canceled_at                     | [Time](/reference/enums#formats) | Read         | Rental's canceled time if canceled.
+{: class="table table-bordered"}
+  </div>
+  <div class="tab-pane" id="rentals_read" markdown="1">
+Name                            | Type     | Read/Write | Description
+--------------------------------|----------|------------|------------
+id                              | Integer  | Read       | Rental contact's id.
+contact_id                      | Integer  | Read       | Contact's id.
+--------------------------------|----------|------------|--------------
+kind                            | [String](/reference/enums#rental-contact-kinds) | Read | Rental contact's kind
+roles                           | [Array of Strings](/reference/enums#rental-contact-roles) | Read | Rental contact's roles
+--------------------------------|----------|------------|--------------
+created_at                      | [Time](/reference/enums#formats) | Read         | Rental's create time.
+updated_at                      | [Time](/reference/enums#formats) | Read         | Rental's update time.
+canceled_at                     | [Time](/reference/enums#formats) | Read         | Rental's canceled time if canceled.
+{: class="table table-bordered"}
+  </div>
+  <div class="tab-pane" id="rentals_write" markdown="1">
+Name                            | Type     | Read/Write | Description
+--------------------------------|----------|------------|------------
+id                              | Integer  | Read       | Rental's id.
+contact_id                      | Integer  | Write      | Contact's id.
+--------------------------------|----------|------------|--------------
+kind                            | [String](/reference/enums#rental-contact-kinds) | Read/Write | Rental contact's kind
+roles                           | [Array of Strings](/reference/enums#rental-contact-roles) | Read/Write | Rental contact's roles
+--------------------------------|----------|------------|--------------
+created_at                      | [Time](/reference/enums#formats) | Read         | Rental's create time.
+updated_at                      | [Time](/reference/enums#formats) | Read         | Rental's update time.
+canceled_at                     | [Time](/reference/enums#formats) | Read         | Rental's canceled time if canceled.
+{: class="table table-bordered"}
+  </div>
+</div>
+
+## List rental contacts
+
+List all rental contacts for a given account(s).
+
+~~~
+GET /rental_contacts
```

Comment:
should be prefixed by /api/v3, same everywhere

Reply:
actually this is the same as other endpoints have. Eg: http://developers.bookingsync.com/reference/endpoints/rentals/#list-rentals

Reply:
Hmm, ok, so let's keep it like this for now